libfreerdp/crypto/crypto.c

```diff
@@ -145,14 +146,14 @@ CryptoHmac crypto_hmac_new(void)
 	return hmac;
 }
 
-void crypto_hmac_sha1_init(CryptoHmac hmac, const BYTE* data, UINT32 length)
+BOOL crypto_hmac_sha1_init(CryptoHmac hmac, const BYTE* data, UINT32 length)
 {
-	HMAC_Init_ex(&hmac->hmac_ctx, data, length, EVP_sha1(), NULL);
+	return HMAC_Init_ex(&hmac->hmac_ctx, data, length, EVP_sha1(), NULL) == 1;
```

Comment:
Correct, in 2008.
See https://github.com/openssl/openssl/commit/87d52468aa600e02326e13f01331e1f3b8602ed0#diff-0a42cda89821d0691aeb6a4a5d124894